filesystem/fs_interface.go

```diff
@@ -0,0 +1,13 @@
+package filesystem
+
+type FileSystem interface {
```

Comment:
I only find the
http://aws.amazon.com/documentation/s3/
it has only restful api.

Xiaoyun

On Sat, Mar 7, 2015 at 5:14 AM, Xiaoyun Wu <xiaoyun.wu@gmail.com> wrote:

> where do I find the official s3 client?
>
>
>
> On Sat, Mar 7, 2015 at 1:52 AM, Xiang Li <notifications@github.com> wrote:
>
>> In filesystem/fs_interface.go
>> <https://github.com/taskgraph/taskgraph/pull/18#discussion_r25962656>:
>>
>> > @@ -0,0 +1,13 @@
>> > +package filesystem
>> > +
>> > +type FileSystem interface {
>>
>> I mean we need to learn from others, not create our own from the very
>> beginning.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/taskgraph/taskgraph/pull/18/files#r25962656>.
>>
>
>